src/routes/q/[slug]/+page.svelte

```
@@ -61,6 +62,10 @@
 								selectedClass = event.selectedClass;
 								selectedUnit = event.selectedUnit;
 							}
+							if (event.type === 'quiz-exit') {
+								requestVersion = 0;
+								void goto(page.url.pathname);
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)